view/frontend/web/js/model/checkout.js

```diff
@@ -181,6 +183,13 @@ function(
         });
       }
 
+      if ('undefined' !== typeof MyParcelConfig && MyParcelConfig.hasOwnProperty('methods')) {
+        MyParcelConfig.methods.forEach(function(code) {
+          var el = document.getElementById('label_method_' + code + '_' + code);
+          if (el && (el = el.parentNode)) el.parentNode.removeChild(el);
```

Comment:
waarom deze assignment in een if? leest ontzettend moeilijk. graag ook altijd een block `{...}` gebruiken